blog-server-api/src/endpoints/create_post/handler.rs

```diff
@@ -0,0 +1,48 @@
+use super::request_content::CreatePostRequestContent;
+use super::response_content_failure::CreatePostContentFailure;
+use super::response_content_failure::CreatePostContentFailure::*;
+use super::response_content_success::CreatePostContentSuccess;
+use crate::entities::CreatePost;
+use crate::extensions::Resolve;
+use blog_server_services::traits::post_service::PostService;
+use screw_api::request::ApiRequest;
+use screw_api::response::ApiResponse;
+use screw_components::dyn_result::DResult;
+use std::sync::Arc;
+
+async fn handler(
+    new_post_data: DResult<CreatePost>,
+    post_service: Arc<Box<dyn PostService>>,
+) -> Result<CreatePostContentSuccess, CreatePostContentFailure> {
+    let base_post = new_post_data
+        .map_err(|e| ValidationError {
+            reason: e.to_string(),
+        })?
+        .into(1);
```

Comment:
[QUESTION] 🤡 WAT?

Reply:
just simplification for db related processes testing. Now i added author id for request body which is also simplification for now. (TODO placed). 

If you pass request with unexisting author we go straight to the database error 🤡🤡🤡🤡🤡